airbyte-integrations/connectors/destination-redshift/src/test/java/io/airbyte/integrations/destination/redshift/RedshiftDestinationTest.java

```diff
@@ -18,22 +20,23 @@
   private static final ObjectMapper mapper = MoreMappers.initMapper();
 
   @Test
-  @DisplayName("When given S3 credentials should use COPY")
-  public void useCopyStrategyTest() {
+  @DisplayName("When given S3 credentials should use COPY with SUPER Datatype")
+  public void useCopyStrategyTestWithSuperTmpTable() {
     final var stubConfig = mapper.createObjectNode();
     stubConfig.put("s3_bucket_name", "fake-bucket");
     stubConfig.put("s3_bucket_region", "fake-region");
     stubConfig.put("access_key_id", "test");
     stubConfig.put("secret_access_key", "test key");
+    stubConfig.put("use_super_redshift_type", "true");
```

Comment:
why do we have it here ? as I understand we removed  this option "use_super_redshift_type" ?

Reply:
Thanks, removed